client/client.go

```diff
@@ -85,14 +85,39 @@ func Configure(logger hclog.Logger, config interface{}) (schema.ClientMeta, erro
 		options = append(options, option.WithCredentialsJSON(serviceAccountKeyJSON))
 	}
 
-	var err error
-	if len(providerConfig.ProjectIDs) == 0 {
```

Comment:
I understand you deprecated projectFilter but not ProjectIds?

Why not support the previous filter as well but make it mutually exclusive with Folder arguments?

i'll let @bbernays chime his thoughts here as well.

Reply:
We should keep the `projectIds` field for 2 reasons:
1. Backwards compatibility 
2. Not everyone uses Folders

Reply:
previous filter is not there in the `v3` api, you'd have to use the `Projects.Search` call (instead of `Projects.List`)
I can support it and keep appending all projects, but is it used?